services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -191,23 +212,33 @@ async def start_study_job(
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
     director2_api: Annotated[DirectorV2Api, Depends(get_api_client(DirectorV2Api))],
     cluster_id: ClusterID | None = None,
-) -> JobStatus:
+) -> JobStatus | JSONResponse:
     job_name = _compose_job_resource_name(study_id, job_id)
     with log_context(_logger, logging.DEBUG, f"Starting Job '{job_name}'"):
-        await start_project(
-            request=request,
-            job_id=job_id,
-            expected_job_name=job_name,
-            webserver_api=webserver_api,
-            cluster_id=cluster_id,
-        )
-        job_status: JobStatus = await inspect_study_job(
+        try:
+            await start_project(
+                request=request,
+                job_id=job_id,
+                expected_job_name=job_name,
+                webserver_api=webserver_api,
+                cluster_id=cluster_id,
+            )
+        except ProjectAlreadyStartedError:
+            job_status: JobStatus = await inspect_study_job(
+                study_id=study_id,
+                job_id=job_id,
+                user_id=user_id,
+                director2_api=director2_api,
+            )
+            return JSONResponse(
+                content=jsonable_encoder(job_status), status_code=status.HTTP_200_OK
+            )
+        return await inspect_study_job(
```

Comment:
if possible, try to keep `return  job_status` since it is more readable than just `return await inspect-study_job`

Reply:
done